README.md

```diff
@@ -65,3 +65,12 @@ Generate product categories that are all child terms of an existing product cate
 
 Generate terms in the Product Tags taxonomy based on the number of terms parameter.
 - `wp wc generate terms product_tag <nr of terms>`
+
+## Development
+
+Requirements
+
+* Node.js v14+
+* Composer v2+
+
+Run `npm run setup` to get started. This will install a pre-commit Git hook that will lint changes to PHP files before they are committed.
```

Comment:
Can we add `nvm use` here and add nvm to the requirements.

Reply:
Good idea c737ed4